pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "great_expectations_cloud"
-version = "0.0.17"
+version = "0.0.18.dev0"
```

Should this be a normal version not a dev version?
```suggestion
version = "0.0.18"
```

---

Thanks for catching this. I just went with the `poetry` output... I think `0.0.18` is correct.

---

Thanks for changing. I think there is an option to the invoke command for a standard version bump. Might be `--standard` https://github.com/great-expectations/cloud?tab=readme-ov-file#release-to-pypi-and-docker